app/controllers/api/room_types_controller.rb

```diff
@@ -0,0 +1,8 @@
+module Api
+  class RoomTypesController < ApplicationController
+    def index
+      @room_types = RoomType.all
+      render json: @room_types
+    end
+  end
+end
```

Comment:
- [ ] Please add an endpoint for room_types in `routes.rb`.  👍

Reply:
I have added it on the second PR #30